_includes/header.html

```diff
@@ -1,9 +1,15 @@
 <header>
   <div class="pure-g">
-    <div class="pure-u-1 pure-u-lg-1-5">
-        <a href="{{ site.baseurl }}/">
-          <img class="nav-icon" src={{ site.logo }} alt="MyPaint" />
-        </a>
+    <div class="logo pure-u-1 pure-u-lg-1-5">
+      <a href="{{ site.url }}">
```

Comment:
it does seem to work as intended on the original site, but not in my dev environment. weiiiird. I'll change it back for now.